src/test/java/ai/rev/speechtotext/integration/SubmitJobTest.java

```diff
@@ -148,6 +148,7 @@ private RevAiJobOptions getJobOptions() {
     revAiJobOptions.setCallbackUrl("https://example.com");
     revAiJobOptions.setSpeakerChannelsCount(null);
     revAiJobOptions.setDeleteAfterSeconds(0);
+    revAiJobOptions.setLanguage("en");
```

Comment:
Looks like we are only testing english transcription. @amikofalvy I noticed that we aren't testing the other languages in the other SDKs either. What are your thoughts around including integration tests for other languages?

Are we relying on the [revai api tests](https://github.com/revdotcom/revai-tests/blob/master/src/test/java/com/revai/api/submission/RevAiForeignLanguagesSubmissionValidation.java) to catch those?

Reply:
The SDK integration tests are testing the SDK's ability to interact with the API. As you point out, the integration test suite for the API covers the testing for the other languages.

Reply:
I don't see a reason to have duplicate coverage which performs real transcription for foreign languages. It will not provide additional benefit and it will cost additional dollars.